spec/services/weather_man_spec.rb

```diff
@@ -36,64 +36,66 @@
 
   describe ".cache_key" do
     it "returns string that includes zip code and date" do
-      key = WeatherMan.key_for(10000, DateTime.parse("October 1, 2015"))
+      key = WeatherMan.key_for(DateTime.parse("October 1, 2015"), 10000)
       expect(key).to include '10000'
       expect(key).to include '2015-10-01'
     end
   end
 
   describe ".outdoor_temp_for" do
-    it "only pulls from the NOAA central park weather station", :vcr do
-      time = Time.zone.parse("Feb 20, 2015 at 8am")
-      response = {
-        "history" => {
-          "observations" => [
-            {
-              "date" => {
-                "hour" => "8"
-              },
-              "tempi" => 30
-            }
-          ]
-        }
-      }
+    let(:empty_response) { { "history" => { "observations" => [] } } }
+    let(:obs_at_30) { { "date" => { "hour" => "08" }, "tempi" => 30 } }
+    let(:obs_at_45) { { "date" => { "hour" => "08" }, "tempi" => 45 } }
+    let(:response_at_30) { { "history" => { "observations" => [obs_at_30] } } }
+    let(:response_at_45) { { "history" => { "observations" => [obs_at_45] } } }
+    let(:eight_am) { Time.zone.parse("Feb 20, 2015 at 8am") }
+    let(:two_pm) { Time.zone.parse("Feb 20, 2015 at 2pm") }
+    let(:eight_pm) { Time.zone.parse("Feb 20, 2015 at 8pm") }
+    let(:time) { Time.zone.parse("March 1, 2015 at 8am") }
+    let(:location) { 'knyc' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.